rfcs/0001-incentivization-transactions.md

```diff
@@ -0,0 +1,66 @@
+# Summary
+
+[summary]: #summary
+
+Provide a mechanism to pay users for having a verified identity and continuously adding data to their data host.
```

Comment:
I know technically it's all the same, but depending on the audience it might make sense to clarify that "user" might also mean "a user-facing application"

Reply:
I don't think that's necessary to include here. I also want to keep the focus on the people using the protocol as much as possible.